### PR TITLE
feat: add PhysicalAddress to Contact

### PR DESCRIPTION
**Pull request by theyokohamalife**
Part of #87

# What changed
1. Adds the `PhysicalAddress` type to `Facility`
2. Creates a migration to add the PhysicalAddress columns
3. Updates the seeds to include the Ja fields for PhysicalAddress
4. Adds query and resolver functions for getting a PhysicalAddress by Id or a list of PhysicalAddresses

# Testing instructions
1. Run `docker compose up`
6. Run `yarn dev`
7. Use the following queries in Apollo server: 
(See "Testing GraphQL" section in README for more detailed steps)

## Test 1: Query PhysicalAddress in Facilities
Operations:
```
query Query {
  facilities {
    physicalAddress {
      postalCode
      addressLine1Ja
      addressLine1En
    }
  }
}
```
## Test 2: Query all PhysicalAddresses
Operations:
```
query Query {
  physicalAddresses {
    addressLine1En
    addressLine1Ja
  }
}
```

## Test 3: Query PhysicalAddress by Id
Operations:
```
query Query($physicalAddressId: ID!) {
  physicalAddress(id: $physicalAddressId) {
    id
    addressLine1En
    addressLine1Ja
  }
}
```

Variables:
```
{
  "physicalAddressId": "2"
}
```

### CODE REVIEW

File: prisma/schema.prisma [hunk 1]
```diff
@@ -2,14 +2,14 @@
 // learn more about it in the docs: https://pris.ly/d/prisma-schema
 
 generator client {
-    provider      = "prisma-client-js"
-    binaryTargets = ["native", "debian-openssl-1.1.x"]
+  provider      = "prisma-client-js"
```
Comment (theyokohamalife) [inline prisma/schema.prisma:5]: I'm guessing this file wasn't formatted?

File: src/services/facilityService.ts [hunk 2]
```diff
@@ -1,21 +1,24 @@
...
     return {
         id: String(input.id),
-        nameJa: input.nameJa,
         nameEn: input.nameEn,
+        nameJa: input.nameJa,
         contact: input.contact,
+        physicalAddress: input.physicalAddress,
         healthcareProfessionals: []
-    } as Facility
+    } as unknown as Facility
```
Comment (theyokohamalife) [inline src/services/facilityService.ts:21]: Could use some input on how to handle this better. Without the `unknown` I get an error about the physicalAddress.id type not matching.

File: prisma/schema.prisma (hunk 1 above)
Comment (ann-kilzer) [reply prisma/schema.prisma:5]: Some of the linting may have changed. Please make a separate pr to fix the style to separate logic and formatting changes

**Review by ann-kilzer — APPROVED** [review]
👍🏻 great work

**Pull request merged**